src/cpu/aarch64/jit_primitive_conf.hpp

```diff
@@ -0,0 +1,723 @@
+    /* bf16 data-type for output */
+    data_type_t dst_dt;
+    data_type_t src_dt;
+    /* bf16 weights update */
+    data_type_t wei_dt;
+    data_type_t dsrc_dt;
+    data_type_t dwei_dt;
+    bool expl_bcast;
+    bool large_spatial, large_w_filter;
+    int is_oc_scale;
+    int max_regs_ur; // maximum accumulation registers
+    // dw conv
+    int nb_ch, ch_block, nb_ch_blocking;
+    bool is_depthwise, is_fast_depthwise, is_resrc_depthwise;
+    int aligned_threads;
+    // large spatial
+    int oh_blk_size;
+    // s8s8 convolution
+    bool signed_input;
+    bool need_saturation;
+    float wei_adj_scale;
+    // zero-point compensation
+    bool src_zero_point;
+    bool dst_zero_point;
+    bool zp_src_is_common; // common, otherwise (TODO) per-channel
+
+    bool uses_permw_transposition;
+    bool transpose_src;
+    bool transpose_dst;
+    int ic_block_step;
+
+    cpu_isa_t isa;
+    // bf16 bwdw conv
+    int tr_ow;
+    bool is_hw_transp; // spatial dim height-width transposed
+    int spatial_blk_size; // Height/depth block size inside the driver
+    bool global_transpose; // diff_dst & src tensors are transposed in one go
+    bool use_nt_stores_ddst; // Use non temporal stores in diff_dst transform
+
+    // Needed for Intel(R) Advanced Matrix Extensions (Intel(R) AMX) kernels
+    bool is_nspc; // activations in nwc, nhwc, or ndhwc layout
+    bool is_relo; // reduced lowering optimization
+    int nreduce; // used with is_relo
+    bool is_pbuffer_strided; // does pbuffer have strided sectors?
+    int n_stride_sets; // number of stride sectors (or sets) in pbuffer
+    int kw_step; // usually stride_w, unless !is_pbuffer_strided
+    int kw_per_tile; // mostly for 1st convs
+    int ic_block_int, ic_block_int_np;
+    int nb_ic_int;
+    int nb_oh_blocking;
+
+    int full_tile_width;
+    int max_tiles;
+    int tile_width;
+    int tile_tail;
+    int oh_per_tile;
+    int ow_blocks;
+
+    int per_one_pstore;
+
+    int inp_buffer_size;
+    int wei_buffer_size;
+    int wsp_buffer_size;
+
+    int nb_os;
+    int nb_os_blocking;
+    int nb_os2_blocking;
+    int os_tail;
+    int os_blocked;
+    int max_width;
+};
+
+// calculates filter size taking into account dilation
+inline int calculate_extended_filter_size(int filter_size, int dilation) {
+    return (filter_size - 1) * (dilation + 1) + 1;
+}
+
+inline int calculate_end_padding(int start_padding, int dst_size, int src_size,
+        int spatial_stride, int dilated_filter_size) {
+    return (dst_size - 1) * spatial_stride + dilated_filter_size
+            - (src_size + start_padding);
+}
+
+inline status_t init_tag(format_tag_t &tag, const memory_desc_wrapper &mdw,
+        const format_tag_t &tag_value) {
+    if (mdw.format_kind() == format_kind::any) return status::unimplemented;
+
+    tag = mdw.matches_one_of_tag(tag_value);
+    return tag == tag_value ? status::success : status::unimplemented;
+}
+
+struct jit_conv_conf_2x3_wino_t {
```

Comment:
I was wondering if there is any plan to support Winograd convolution, otherwise we can remove this code.

Reply:
I left it because I wanted to support Winograd convolution in the near future, but I will delete it.
I will add it when I need it.